committee-steering/governance/working-group-formation-process.md

```diff
@@ -0,0 +1,87 @@
+# Kubernetes Working Group Formation Process - Proposal
+
+Draft 1.0 // Jaice Singer DuMars // June, 2018
+
+## Process Overview and Motivations
+As opposed to “Birds of a Feather” or discussion groups, Working Groups provide a more formal avenue for disparate
+groups to intellectually collaborate around a common problem, craft a balanced position, and disband. Because they
+represent the interests of multiple special interest groups, they are a vehicle for consensus building, not
+construction. By design, the code under discussion resides in either multiple parts of the project, or is in the
+process of being extracted to an external repository.
+
+Because a Working Group is an acknowledged part of the Kubernetes governance model, a process is necessary for
```

Comment:
This sentence is wordy and long.

"Because a working group is an official part of the Kubernetes project it is subject to steering committee oversight over its formation, disbanding and authority."

Reply:
Done